app/model_config.py

```diff
@@ -23,29 +23,32 @@
 #                         ├── variables.data-00000-of-00001
 #                         └── variables.index
 
-
-IN = {
-    "bucket_name": "bert_classification_models",
-    "model_name": "UPR_2percent_ps0",
-    "instance_name": "1573031002",
-    "saved_model": {
-        "name": "saved_model.pb",
-        "path": "multilabel/{model}/saved_model/{instance}",
-    },
-    "variables": {
-        "name": "*",
-        # TODO: come up with a convention on handling recursive copying in the
-        # input configuration.
-        "path": "multilabel/{model}/saved_model/{instance}/variables",
-    },
-    "vocab": {
-        "name": "label.vocab",
-        "path": "multilabel/{model}/saved_model",
-    },
-}
+# TODO: Stop hardcoding this and provide sane default values for the CLI.
```

Comment:
is this done by ./static/model_config.json?

Reply:
yes